### ISSUE TITLE
After-steps aren't being reported when running in the cloud

### ISSUE THREAD

**Issue by flenter** [wercker/wercker#46]
It happens when the run fails.

Here's an example wercker.yml:

``` yaml
box: node:4.1.2
build:
  steps:
    - script:
        name: run test
        code: |
          fail_now
  after-steps:
      - script:
          name: after
          code: |
              echo "after"
              echo $WERCKER_RESULT
              echo "---"
```

I am unsure whether this is an issue for reporter-client specific or wercker/wercker.

**Comment by flenter** [comment]
<img width="953" alt="screen shot 2016-02-03 at 17 18 14" src="https://cloud.githubusercontent.com/assets/473804/12788712/f668ab30-ca9a-11e5-8eae-f06f21570069.png">

**Comment by hatchan** [comment]
I can confirm that the after steps are being run, but they are not being reported

**Comment by termie** [comment]
closed by #44 #48 and #51
